README.md

```diff
@@ -71,25 +69,24 @@ Example responses:
 ```
 HTTP/1.1 201 Created
 ```
-The above response would indicate a new object has been created (no existing object with the given UUID was found).
+The above response would indicate a new object has been created (no existing object with the given ID was found).
 
 ```
 HTTP/1.1 200 OK
 ```
-The above response would indicate an existing object has been updated (existing object with the given UUID was found).  Note that a PUT operation on an existing record may also result in "403 Forbidden" error.
+The above response would indicate an existing object has been updated (existing object with the given ID was found).  Note that a PUT operation on an existing record may also result in "403 Forbidden" error.
 
 
 ### List - `store/{bucket}` endpoint
 This endpoint allows for `GET` to retrieve all the items in the bucket organized by the id.
 
-An example response will look like where "earth" is the id of the item. An optional header `X-Midt-Owner` can be sent with the request.  If supplied, only items with secrets matching the supplied value will be returned in the list. If items from all owners are desired, use the `X-Midt-Admin-Token` header. 
+An example response will look like where "earth" is the id of the item. An optional header `X-Midt-Owner` can be sent with the request.  If supplied, only items with secrets matching the supplied value will be returned in the list. Only authorized requests can retrieve items from all owners.
```

Comment:
`earth` is an invalid id reference.  Change to `7e8c5f378b4addbaebc70897c4478cca06009e3e360208ebd073dbee4b3774e7`

Reply:
Updated.